docs/diagrams/commands/ViewProfileCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+!include ../style.puml
+
+box Logic LOGIC_COLOR_T1
+Participant ":LogicManager" as logicManager LOGIC_COLOR_T3
+Participant ":NuSchedulerParser" as nuSchedulerParser LOGIC_COLOR_T3
+Participant "p:ViewProfileCommand" as viewProfileCommand LOGIC_COLOR_T3
+Participant "r:CommandResult" as commandResult LOGIC_COLOR_T3
+endBox
+
+box Model MODEL_COLOR_T1
+Participant "model:Model" as model MODEL_COLOR_T3
+Participant "filteredProfiles:FilteredList" as filteredList MODEL_COLOR_T3
+endBox
+
+-> logicManager ++: execute("profile -v")
+logicManager -> nuSchedulerParser ++: parseCommand
+return p
```

Comment:
```suggestion
create viewProfileCommand
return p
nuSchedulerParser -[hidden]-> viewProfileCommand
```

Reply:
Ohh I see. It is a clever use of the hidden! 👍  Thanks!